src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -527,6 +546,21 @@ public DirContext bind(String principalName, String password, List<SocketInfo> l
             throw new BadCredentialsException("Either no such user '"+principalName+"' or incorrect password", error);
         }
 
+        /**
+         * Binds to the server using the specified username/password.
+         * <p>
+         * In a real deployment, often there are servers that don't respond or
+         * otherwise broken, so try all the servers.
+         */
+        @Deprecated
+        public DirContext bind(String principalName, String password, List<SocketInfo> ldapServers) {
+            Hashtable<String, String> props = new Hashtable<String, String>();
+            props.put(Context.REFERRAL, "follow");
+            props.put("java.naming.ldap.attributes.binary", "tokenGroups objectSid");
```

Comment:
you are adding these three anyway in the `bind(String principalName, String password, List<SocketInfo> ldapServers, Hashtable<String, String> props)` method.